### PR TITLE
Feature/project setup

### PR DESCRIPTION
**Pull request by Mohamed-js**
Setup project, linters and printing "Hello, world!"

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've done well so far in this milestone.
However, some changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] `main.rb` should be an executable file ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
- `#!/usr/bin/env` ruby is the top line
- puts "Hello world" in the following line
- naming the file to only main is allowed





* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by ebeagusamuel — APPROVED** [review]
# Project Approved :100: 
#### Hey there :wave: , this milestone has been approved. You can start working on the next milestone.
### Good luck
slack(@ebeagusamuel)

**Pull request merged**
